tests/test_core.py

```diff
@@ -0,0 +1,39 @@
+from roslibpy import Header
+from roslibpy import Time
+
+REF_FLOAT_SECS_TIME = 1610122759.677662
+
+
+def test_time_from_sec_based_on_time_module():
+    t = Time.from_sec(REF_FLOAT_SECS_TIME)
+    assert t.secs == 1610122759
+    assert t.nsecs == 677661895
+
+
+def test_to_nsec():
+    t = Time.from_sec(REF_FLOAT_SECS_TIME)
+    assert t.to_nsec() == 1610122759677661895
+
+
+def test_to_sec():
+    t = Time.from_sec(REF_FLOAT_SECS_TIME)
+    assert t.to_sec() == REF_FLOAT_SECS_TIME
+
+
+def test_is_zero():
```

Comment:
I would also add some negative tests here, eg `assert not Time(1, 0).is_zero()`.

Reply:
added on https://github.com/gramaziokohler/roslibpy/pull/72/commits/a08dea94559d809a9b5edaa155542ac26e58e81f